src/Appwrite/Extend/Exception.php

```diff
@@ -84,6 +84,8 @@ class Exception extends \Exception
     public const USER_OAUTH2_BAD_REQUEST           = 'user_oauth2_bad_request';
     public const USER_OAUTH2_UNAUTHORIZED          = 'user_oauth2_unauthorized';
     public const USER_OAUTH2_PROVIDER_ERROR        = 'user_oauth2_provider_error';
+    public const USER_EMAIL_ALREADY_VERIFIED        = 'user_email_alread_verified';
+    public const USER_PHONE_ALREADY_VERIFIED        = 'user_phone_already_verified';
```

Comment:
Let's make sure the alignment is consistent.

```suggestion
    public const USER_EMAIL_ALREADY_VERIFIED       = 'user_email_alread_verified';
    public const USER_PHONE_ALREADY_VERIFIED       = 'user_phone_already_verified';
```

Reply:
Sure will fix this